runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -129,6 +129,8 @@ typedef struct memEntry (memTab_t)[MAX_MEM_REGIONS];
 static memTab_t memTab;
 static memTab_t* memTabMap;
 
+#define AM_MAX_MSG_SIZE (10 * 10240) // TODO: safe (?), but awfully large
```

Comment:
Make an issue for this?

Reply:
Updated the complete comm=ofi [epic](https://github.com/chapel-lang/chapel/issues/12264).